DeusEx/Classes/DXRFixup.uc

```diff
@@ -139,6 +141,51 @@ function NYC1_FirstEntry()
     }
 }
 
+function Airfield_FirstEntry()
```

Comment:
I see another place that could use a button

run `open 03_NYC_BROOKLYNBRIDGESTATION`, turn around, and look at where the telephone booth is supposed to come down

it's not really a softlock cause I think you should be able to progress no matter what anyways?

Reply:
The phone booth elevator is actually in the Battery Park map, so it isn't a concern.  Since you always start in Battery Park, you'll always be able to progress downwards.  In theory, we could maybe still put a button in Battery Park though.